Solutions/Marain.TenantManagement.Specs/Data/ServiceManifests/OperationsServiceManifest.json

```diff
@@ -1,8 +1,11 @@
 {
   "$schema": "./ServiceManifestSchema.json",
   "contentType": "application/vnd.marain.tenancy.servicemanifests.servicemanifest",
+  "wellKnownTenantGuid": "4f522924-b6e7-48cc-a265-a307407ec858", // 3633754ac4c9be44b55bfe791b1780f12429524fe7b6cc48a265a307407ec858
   "serviceName": "Operations v1",
-  "dependsOnServiceNames": [ "FooBar v1" ],
+  "dependsOnServiceTenantIds": [
+    "3633754ac4c9be44b55bfe791b1780f11ea674ff6030d340bfec60dfe12febac" // FooBar v1
```

Comment:
The presence of this comment demonstrates my earlier point. 😀

Reply:
If we unmangled and alllowed non-GUID IDs, then we would be in good shape.